roles/infra/tasks/main.yml

```diff
@@ -46,6 +46,12 @@
     content: "{{ _terraform_backend_config | to_json }}"
     dest: "{{ terraform_project_path }}/backend_config.json"
 
+- name: Set terraform to rebuild patched appliances
+  include_tasks: rebuild.yml
+  when:
+    - terraform_rebuild_appliance == 'patch'
```

Comment:
Can we just make this a boolean please? I can't see why it would need to have more than yes/no semantics. Also, can this role doesn't really know the "appliance" terminology so maybe call this `terraform_rebuild_nodes`?